cypress/e2e/Navbar.cy.js

```diff
@@ -0,0 +1,46 @@
+describe('Test the Navabr routing works', () => {
+  it('Successfully navigates to the Home Page upon clicking Home', () => {
```

Comment:
Try and use 'should' when starting your bbd tests. You utilised such concepts in the above tests